pkg/sources/fake/fake.go

```diff
@@ -6,15 +6,36 @@ import (
 
 	"telemetry/pkg/msgs"
 	"telemetry/pkg/pubsub"
+	"telemetry/pkg/util/randutil"
 )
 
 const (
 	// interval between fake data readings
-	readInterval = 20 * time.Millisecond
+	readInterval = 200 * time.Millisecond
+
+	// TODO(karlding): change these to the actual values
+	rightTurnOn     = 1
```

Comment:
So it seems simpler and perhaps even more logical to have rightTurn and similar on/off messages be a single id and the value of the message be a boolean denoting whether to turn it on or off? This seems a lot cleaner to me and applies to 6 categories so far as I can tell? This saves us 3 messages and allows you to wrap a lot of the logic in functions to reduce the number of bus.Publish calls? WDYT?